src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,82 @@
-/* eslint-disable max-len */
-import React from 'react';
+import { useMemo } from 'react';
+import { TodoItem } from '../TodoItem';
+import { useAppSelector } from '../../app/hooks';
+import { Todo } from '../../types/Todo';
 
 export const TodoList: React.FC = () => {
+  const query = useAppSelector(state => state.filter.query);
+  const status = useAppSelector(state => state.filter.status);
+  const todos = useAppSelector(state => state.todos);
+
+  const getFilteredTodos = (todosList: Todo[]) => {
+    if (!todosList.length) {
+      return [];
+    }
```

Comment:
BTW. Method `filter()` returns empty array for empty arrays 
![image](https://user-images.githubusercontent.com/70009205/211381416-0ecccdd8-445b-46f3-81da-8af13f154718.png)

Reply:
Done ✅